language/move-prover/tests/sources/functional/inline-lambda.exp

```diff
@@ -0,0 +1,45 @@
+Move prover returns: exiting with verification errors
+error: unknown assertion failed
+  ┌─ tests/sources/functional/inline-lambda.move:5:13
```

Comment:
Unknown assertion? Are these errors expected?

Reply:
Yes, assertion failures in `inline spec` somehow got the name of `unknown assertion`. This should be the case even before inline function exists.

These errors are indeed expected and show that after inlining and lambda lifting, assertions in the "inline spec" can fail.

Reply:
This is something we need to fix, but not in this PR. It seems the PropKind is unknown?

https://github.com/move-language/move/blob/4ef43b34cf2f926a5b9d5213c0cb0f05653b1e38/language/move-prover/boogie-backend/src/bytecode_translator.rs#L992